### PR TITLE
Updated publications list

### PR DESCRIPTION
**Pull request by jdgarciauc3m**
Updated the list of publications in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,20 @@ Please cite this publication in any work using our library:
...
 
+* **Parallelizing and optimizing LHCb-Kalman for Intel Xeon Phi KNL processors**. Placido Fernandez, David del Rio, Manuel .F. Dolz, Javier Fernández, Omar Awile, J. Daniel Garcia *PDP 2018*
+
+* **Supporting Advanced Patterns in GrPPI: a Generic Parallel Pattern Interface**. David del Rio, Manuel F. Dolz, Javier Fernandez, and J. Daneil Garcia. *Auto-DaSP 2017 (Euro-Par 2017)*. Santiago de Compostela, Spain. 28/8-1/9/2017. pp. 55-67. DOI[10.1007/978-3-319-75178-8_5](https://doi.org/10.1007/978-3-319-75178-8_5)
+
+* **Probabilistic-Based Selection of Alternate Implementations for Heterogeneous Platforms**. J. Fernandez, A. Sanchez, D. del Río, M. F. Dolz, J. D Garcia. **ICA3PP 2017**. 2017. Helsinki, Finalnd. 21/23/8/2017. pp. 74-87. DOI: [10.1007/978-3-319-49583-5_5](https://doi.org/10.1007/978-3-319-49583-5_5)
```
Comment (mdolz) [inline README.md]: This publication is not related to GrPPI, please drop it from this list.

File: README.md [hunk 2]
```diff
@@ -57,11 +57,20 @@ Please cite this publication in any work using our library:
 
 ### Other references
 
-* **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
 
+* **Parallelizing and optimizing LHCb-Kalman for Intel Xeon Phi KNL processors**. Placido Fernandez, David del Rio, Manuel .F. Dolz, Javier Fernández, Omar Awile, J. Daniel Garcia *PDP 2018*
+
+* **Supporting Advanced Patterns in GrPPI: a Generic Parallel Pattern Interface**. David del Rio, Manuel F. Dolz, Javier Fernandez, and J. Daneil Garcia. *Auto-DaSP 2017 (Euro-Par 2017)*. Santiago de Compostela, Spain. 28/8-1/9/2017. pp. 55-67. DOI[10.1007/978-3-319-75178-8_5](https://doi.org/10.1007/978-3-319-75178-8_5)
```
Comment (mdolz) [inline README.md]: Replace "DOI[" by DOI: [".

Comment (mdolz) [reply README.md]: Replace "Daneil" by "Daniel".

File: README.md [hunk 3]
```diff
@@ -57,11 +57,20 @@ Please cite this publication in any work using our library:
 
 ### Other references
 
-* **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
 
+* **Parallelizing and optimizing LHCb-Kalman for Intel Xeon Phi KNL processors**. Placido Fernandez, David del Rio, Manuel .F. Dolz, Javier Fernández, Omar Awile, J. Daniel Garcia *PDP 2018*
```
Comment (mdolz) [inline README.md]: Replace "Placido Fernandez" by "Plácido Fernández".

File: README.md [hunk 4]
```diff
@@ -57,11 +57,20 @@ Please cite this publication in any work using our library:
...
+
+* **Probabilistic-Based Selection of Alternate Implementations for Heterogeneous Platforms**. J. Fernandez, A. Sanchez, D. del Río, M. F. Dolz, J. D Garcia. **ICA3PP 2017**. 2017. Helsinki, Finalnd. 21/23/8/2017. pp. 74-87. DOI: [10.1007/978-3-319-49583-5_5](https://doi.org/10.1007/978-3-319-49583-5_5)
+
+* **Finding parallel patterns through static analysis in C++ applications**. David del Río, Manuel F. Dolz, Luís M. Sanchez, J. Daniel Garcia, Marco Danelutto, and Massimo Torquati. *International Journal of High Performance Computing Applications*. 2017. DOI: [10.1177/1094342017695639](https://doi.org/10.1177/1094342017695639)
+
+* **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
```
Comment (mdolz) [inline README.md]: Homogenize author names: replace "David Del Rio Astorga" by "David del Rio".

**Review by mdolz — CHANGES_REQUESTED** [review]
See comments.

File: README.md (hunk 3 above)
Comment (jdgarciauc3m) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (jdgarciauc3m) [reply README.md]: OK

File: README.md (hunk 1 above)
Comment (jdgarciauc3m) [reply README.md]: Removing

File: README.md (hunk 4 above)
Comment (jdgarciauc3m) [reply README.md]: OK

**Comment by jdgarciauc3m** [comment]
I have addressed comments. Please, double check!

File: README.md [hunk 5]
```diff
@@ -57,11 +57,18 @@ Please cite this publication in any work using our library:
 
 ### Other references
 
-* **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
 
+* **Parallelizing and optimizing LHCb-Kalman for Intel Xeon Phi KNL processors**. Plácido Fernández, David del Río, Manuel .F. Dolz, Javier Fernández, Omar Awile, J. Daniel Garcia *PDP 2018*
+
+* **Supporting Advanced Patterns in GrPPI: a Generic Parallel Pattern Interface**. David del Rio, Manuel F. Dolz, Javier Fernandez, and J. Daniel Garcia. *Auto-DaSP 2017 (Euro-Par 2017)*. Santiago de Compostela, Spain. 28/8-1/9/2017. pp. 55-67. DOI: [10.1007/978-3-319-75178-8_5](https://doi.org/10.1007/978-3-319-75178-8_5)
+
```
Comment (mdolz) [inline README.md]: Replace: "Fernandez" by "Fernández".

Comment (mdolz) [reply README.md]: Replace: "Rio" by "Río".

File: README.md [hunk 6]
```diff
@@ -57,11 +57,18 @@ Please cite this publication in any work using our library:
...
+* **Finding parallel patterns through static analysis in C++ applications**. David del Río, Manuel F. Dolz, Luís M. Sanchez, J. Daniel Garcia, Marco Danelutto, and Massimo Torquati. *International Journal of High Performance Computing Applications*. 2017. DOI: [10.1177/1094342017695639](https://doi.org/10.1177/1094342017695639)
+
+* **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
```
Comment (mdolz) [inline README.md]: Replace: "Del Río" by "del Río".

**Review by mdolz — CHANGES_REQUESTED** [review]
See comments.

**Review by mdolz — APPROVED** [review]
